tests/director/tests/application_templates_subscription_test.go

```diff
@@ -26,6 +26,12 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	RegionValue      = "eu-1"
```

Comment:
I think I've seen this `eu-1` string as part of some of our values.yaml configurations. Maybe it's the `global.defaultTenantRegion` value? In this case it would be better to not hard code this here but rather use an env variable for accessing this value.